fastlane/lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -10,7 +10,10 @@ def self.run(params)
         begin
           path = File.expand_path(params[:path])
           plist = Plist.parse_xml(path)
-          if params[:subkey]
+          if params[:map]
+            stringified_hash = Hash[params[:map].collect { |k, v| [k.to_s, v] }]
+            plist = plist.merge!(stringified_hash)
```

Comment:
Should perhaps return plist to be able to validate test? And some might want it to return the plist? To further update the map?

Reply:
@ohayon WDYT?